diff --git a/node_modules/@rescriptbr/reform/src/ReForm.res b/node_modules/@rescriptbr/reform/src/ReForm.res
index 70eb909..164f466 100644
--- a/node_modules/@rescriptbr/reform/src/ReForm.res
+++ b/node_modules/@rescriptbr/reform/src/ReForm.res
@@ -132,7 +132,7 @@ module Make = (Config: Config) => {
       validateField,
       state,
     }) => {
-      handleChange: handleChange(field),
+      handleChange: handleChange(field, ...),
       error: getFieldError(Field(field)),
       state: getFieldState(Field(field)),
       validate: () => validateField(Field(field)),
